pontoon/base/templates/widgets/heading_info.html

```diff
@@ -130,3 +130,21 @@
   </div>
 </div>
 {% endmacro %}
+
+{% macro item_download_button(locale,slug,class='') %}
```

Comment:
Nits:
* I'd rename `item_download_button` to just `download_button`
* I'd rename `slug` to `project` (to be consistent with locale representing locale **code**).
* Please also use space after comma.